### PR TITLE
Docs on alerts, dashboards and runbooks for Promscale

### PR DESCRIPTION
**Pull request by VineethReddy02**
# Description

[Short summary of why you created this PR]

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Fixes #[insert issue link, if any]

### CODE REVIEW

File: promscale/alerts-dashboards-runbooks.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Alerts, Dashboards and Runbooks
```
Comment (ramonguiu) [inline promscale/alerts-dashboards-runbooks.md:1]:
```suggestion
# Monitoring Promscale
```

File: promscale/page-index/page-index.js [hunk 2]
```diff
@@ -201,34 +231,11 @@ module.exports = [
...
-          },
-          {
-            title: 'Jaeger',
-            href: 'jaeger',
-            tags: ['promscale', 'configure', 'jaeger', 'opentelemetry', 'traces'],
-            keywords: ['Promscale', 'Jaeger'],
-            excerpt: 'Jaeger to visualize data in Promscale',
-          },
-        ],
+        title: "Alerts, Dashboards and Runbooks",
```
Comment (ramonguiu) [inline promscale/page-index/page-index.js:234]:
```suggestion
        title: "Monitoring Promscale",
```

File: promscale/page-index/page-index.js [hunk 3]
```diff
@@ -201,34 +231,11 @@ module.exports = [
...
+        href: "alerts-dashboards-runbooks",
```
Comment (ramonguiu) [inline promscale/page-index/page-index.js:235]: I would change the name of the file and the link here to `monitoring-promscale`

File: promscale/alerts-dashboards-runbooks.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+This sections describes how one can monitor Promscale using dashboards, alerts and
+runbooks to troubleshoot Promscale.
```
Comment (ramonguiu) [inline promscale/alerts-dashboards-runbooks.md:4]: Promscale comes 
```suggestion
Promscale includes a set of out-of-the-box alerts, runbooks and a Grafana dashboard that you can use to monitor it.
```

File: promscale/alerts-dashboards-runbooks.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+<highlight type="note">
+To use Promscale dashboard make sure to configure Promscale as Prometheus data source
```
Comment (ramonguiu) [inline promscale/alerts-dashboards-runbooks.md:7]: We should recommend that people set up a separate Prometheus instance to monitor Promscale. If Promscale goes down it won't be able to alert you that it has gone down. Then they should create a Prometheus data source in Grafana for that Prometheus instance. They also need to ensure the Prometheus instance is scraping the http://<promscale-host>:<promscale-port>/metrics where the default port is 9201

File: promscale/alerts-dashboards-runbooks.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+1.  Open the dashboard published by Promscale org from the [Grafana 
+    community dashboards](grafana-promscale-dashboard), select `Copy ID to Clipboard` to copy the dashboard UID.
+1.  In Grafana UI select `Create` from left side menu bar, navigate to `Import`.
+1.  Now paste the dashbaord UID in `Import via grafana.com` textbox and click 
+    on `Load` to import the dashbaord into Grafana instance.
+1.  On importing the dashboard select the `folder` to which you want to add the imported
+    dashboard.
+1.  Select the data source from which you want the dashboard to query the data.
+    * For Promscale metrics select the `Prometheus` data source as `Promscale-PromQL`.
```
Comment (ramonguiu) [inline promscale/alerts-dashboards-runbooks.md:39]: This should be the data source they created for the separate Prometheus instance

File: promscale/alerts-dashboards-runbooks.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+
+</procedure>
+
+## Alerts
+
+You can configure alerting rules for Promscale to fire alerts 
+when something doesn't work as expected in Promscale.
+
+[Here](promscale-alerting-rules) are the alerting rules for Promscale. These 
+alerting rules can be configured on Prometheus end or on Promscale end to evaluate
```
Comment (ramonguiu) [inline promscale/alerts-dashboards-runbooks.md:49]: Those should be evaluated in the separate Prometheus instance. Again you don't want your monitoring tool to monitor itself

File: promscale/alerts-dashboards-runbooks.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+the alerting rules.
+
+## Runbooks
+
+The runbooks are to troubleshoot the most common issues noticed while running
```
Comment (ramonguiu) [inline promscale/alerts-dashboards-runbooks.md:54]: If I am not mistaken the runbooks are linked in the alert messages that are sent. We should say that.

File: promscale/page-index/page-index.js [hunk 9]
```diff
@@ -201,34 +231,11 @@ module.exports = [
...
+        tags: ["promscale", "prometheus", "alert"],
+        keywords: ["Promscale", "Prometheus", "alert", "Alert Manager"],
+        excerpt: "Learn about alerts, dashboards and runbooks for Promscale",
```
Comment (ramonguiu) [inline promscale/page-index/page-index.js:238]:
```suggestion
        excerpt: "Learn about how to monitor Promscale",
```

**Review by ramonguiu — CHANGES_REQUESTED** [review]

File: promscale/alerts-dashboards-runbooks.md (hunk 8 above)
Comment (VineethReddy02) [reply promscale/alerts-dashboards-runbooks.md:54]: yup!

File: promscale/monitoring-promscale.md [hunk 10]
```diff
@@ -0,0 +1,63 @@
+# Monitoring Promscale
+Promscale includes a set of out-of-the-box alerts, runbooks and a Grafana dashboard that you can use to monitor it.
+
+<highlight type="note">
+To monitor Promscale make sure that Prometheus instance is scraping Promscale
```
Comment (ramonguiu) [inline promscale/monitoring-promscale.md:5]:
```suggestion
To monitor Promscale make sure that a Prometheus instance is scraping Promscale
```

**Review by ramonguiu — APPROVED** [review]

File: promscale/monitoring-promscale.md [hunk 11]
```diff
@@ -1,12 +1,13 @@
-# Alerts, Dashboards and Runbooks
-
-This sections describes how one can monitor Promscale using dashboards, alerts and
-runbooks to troubleshoot Promscale.
+# Monitoring Promscale
+Promscale includes a set of out-of-the-box alerts, runbooks and a Grafana dashboard that you can use to monitor it.
```
Comment (Loquacity) [inline promscale/monitoring-promscale.md:2]:
```suggestion
Promscale includes a set of out-of-the-box alerts, runbooks, and a Grafana dashboard that you can use to monitor it.
```

File: promscale/monitoring-promscale.md [hunk 12]
```diff
@@ -1,12 +1,13 @@
...
 
 <highlight type="note">
-To use Promscale dashboard make sure to configure Promscale as Prometheus data source 
-as mentioned [here](/visualize-data/grafana/#promscale-as-prometheus-datasource) to 
-query the Promscale metrics from promscale.
+To monitor Promscale make sure that a Prometheus instance is scraping Promscale
+http endpoint i.e. default to `9201` port, `/metrics` API. Use Prometheus to
```
Comment (Loquacity) [inline promscale/monitoring-promscale.md:6]:
```suggestion
To monitor Promscale, make sure that a Prometheus instance is scraping the Promscale
HTTP endpoint, which defaults to port `9201` port in the `/metrics` API. Use Prometheus to 
```

Not sure if this is correct. Either way, don't use "ie".

File: promscale/monitoring-promscale.md [hunk 13]
```diff
@@ -1,12 +1,13 @@
...
+# Monitoring Promscale
+Promscale includes a set of out-of-the-box alerts, runbooks and a Grafana dashboard that you can use to monitor it.
 
 <highlight type="note">
-To use Promscale dashboard make sure to configure Promscale as Prometheus data source 
-as mentioned [here](/visualize-data/grafana/#promscale-as-prometheus-datasource) to 
-query the Promscale metrics from promscale.
+To monitor Promscale make sure that a Prometheus instance is scraping Promscale
+http endpoint i.e. default to `9201` port, `/metrics` API. Use Prometheus to 
+evaluate the Promscale alerting rules and also configure Prometheus as data source
```
Comment (Loquacity) [inline promscale/monitoring-promscale.md:7]:
```suggestion
to evaluate the Promscale alerting rules, and configure Prometheus as a data source
```

File: promscale/monitoring-promscale.md [hunk 14]
```diff
@@ -1,12 +1,13 @@
...
+in Grafana to visulaize the Promscale dashboard. This helps to troubleshhot the
```
Comment (Loquacity) [inline promscale/monitoring-promscale.md:8]:
```suggestion
in Grafana to visualize the Promscale dashboard. This helps to troubleshoot the
```

File: promscale/monitoring-promscale.md [hunk 15]
```diff
@@ -1,12 +1,13 @@
...
+issues when Promscale isn't working as expected, again you don't want your monitoring
```
Comment (Loquacity) [inline promscale/monitoring-promscale.md:9]:
```suggestion
issues when Promscale isn't working as expected. You don't want your monitoring
```

File: promscale/monitoring-promscale.md [hunk 16]
```diff
@@ -36,7 +37,9 @@ Import Promscale dashboard by following below instructions
 1.  On importing the dashboard select the `folder` to which you want to add the imported
     dashboard.
 1.  Select the data source from which you want the dashboard to query the data.
-    * For Promscale metrics select the `Prometheus` data source as `Promscale-PromQL`.
+    * To visualize Promscale metrics select the `Prometheus` data source as `Prometheus` query endpoint
```
Comment (Loquacity) [inline promscale/monitoring-promscale.md:40]:
```suggestion
    * To visualize Promscale metrics, select the `Prometheus` data source and `Prometheus` query endpoint.
```

File: promscale/monitoring-promscale.md [hunk 17]
```diff
@@ -36,7 +37,9 @@ Import Promscale dashboard by following below instructions
...
+      we do not want to query Promsale own dashboard using Promscale query endpoint. With this practice
```
Comment (Loquacity) [inline promscale/monitoring-promscale.md:41]:
```suggestion
      You do not want to query Promsale's own dashboard using the Promscale query endpoint.
```

File: promscale/monitoring-promscale.md [hunk 18]
```diff
@@ -36,7 +37,9 @@ Import Promscale dashboard by following below instructions
...
+      we can avoid the single point of failure when Promscale isn't working as expected.
```
Comment (Loquacity) [inline promscale/monitoring-promscale.md:42]:
```suggestion
      This avoids a single point of failure if Promscale isn't working as expected.
```

File: promscale/monitoring-promscale.md [hunk 19]
```diff
@@ -46,13 +49,13 @@ You can configure alerting rules for Promscale to fire alerts
 when something doesn't work as expected in Promscale.
 
 [Here](promscale-alerting-rules) are the alerting rules for Promscale. These 
-alerting rules can be configured on Prometheus end or on Promscale end to evaluate
-the alerting rules.
+alerting rules should be configured on Prometheus end to evaluate the alerting rules.
```
Comment (Loquacity) [inline promscale/monitoring-promscale.md:52]:
```suggestion
alerting rules should be configured on the Prometheus end to evaluate the alerting rules.
```

File: promscale/monitoring-promscale.md [hunk 20]
```diff
@@ -46,13 +49,13 @@ You can configure alerting rules for Promscale to fire alerts
...
 
 ## Runbooks
 
-The runbooks are to troubleshoot the most common issues noticed while running 
-Promscale at scale. The runbooks are published and
+When an alert is fired, the alert message references the relevant runbook
+to troubleshoot and mitigate the issue noticed while running 
+Promscale. These runbooks are published and
```
Comment (Loquacity) [inline promscale/monitoring-promscale.md:58]:
```suggestion
to troubleshoot and mitigate the issue.
These runbooks are published and
```

File: promscale/monitoring-promscale.md [hunk 21]
```diff
@@ -46,13 +49,13 @@ You can configure alerting rules for Promscale to fire alerts
...
 maintained in [Promscale Github repository](promscale-runbooks).
```
Comment (Loquacity) [inline promscale/monitoring-promscale.md:59]:
```suggestion
maintained in the [Promscale Github repository](promscale-runbooks).
```

**Review by Loquacity — CHANGES_REQUESTED** [review]

**Comment by VineethReddy02** [comment]
closing this in favor to #1090

**Pull request closed without merging**